siac/hostcmd.go

```diff
@@ -160,7 +160,8 @@ func hostcmd() {
 	Revenue:             %v
 	Lost Revenue:        %v
 `, filesizeUnits(int64(totalstorage)), filesizeUnits(int64(totalstorage-storageremaining)),
-		price, is.MaxDuration, accept, totalPotentialRevenue, totalRevenue, fm.LostRevenue)
+		price, is.MaxDuration, accept, currencyUnits(totalPotentialRevenue),
+		currencyUnits(totalRevenue), currencyUnits(fm.LostRevenue))
```

Comment:
There's a field missing here which is lost collateral. Lost revenue is stuff that the renter would have paid you but didn't, meaning no real harm done just disappointing, and lost collateral is money you outright threw into the void.